tools/xml2palette/xml2palette.py

```diff
@@ -310,11 +324,11 @@ def process_compounddef(compounddef):
                 direction = pichild[0].attrib.get("direction", "").strip()
             elif pichild.tag == "parameterdescription":
                 if key == "gitrepo":
-                    if pichild[0][0].text is None:
+                    if pichild[0].text is None:
                         print("No gitrepo text")
                         value = ""
                     else:
-                        value = pichild[0][0].text.strip()
+                        value = pichild[0].text.strip()
```

Comment:
I assumed the gitrepo doc for example expands from `\param gitrepo $(GIT_REPO)` to `\param gitrepo https://github.com/ICRAR/EAGLE`

Reply:
It's hard to tell what is correct here, I don't remember why the extra [0] is there.

Let's go with your changes though, and we'll revert if something goes wrong.